src/clj/game/cards-operations.clj

```diff
@@ -552,6 +552,18 @@
     :msg "do 1 net damage"
     :effect (effect (damage eid :net 1 {:card card}))}
 
+   "Observe and Destroy"
+   {:req (req (and (pos? (:tag runner))
```

Comment:
Yes, it initializes at 0 and I'm not aware of anything that would ever drop it completely. Could still add this to be safe.